policybot/mgrs/syncmgr/mgr.go

```diff
@@ -1188,6 +1188,104 @@ func (ss *syncState) handleTestResults() error {
 	return nil
 }
 
+func (ss *syncState) handlePostSubmitTestResults() error {
+	for _, repo := range ss.mgr.reg.Repos() {
+		r, ok := ss.mgr.reg.SingleRecord(refresher.RecordType, repo.OrgAndRepo)
+		if !ok {
+			continue
+		}
+
+		tor := r.(*refresher.TestOutputRecord)
+		g := resultgatherer.TestResultGatherer{
+			Client:           ss.mgr.blobstore,
+			BucketName:       tor.BucketName,
+			PreSubmitPrefix:  tor.PreSubmitTestPath,
+			PostSubmitPrefix: tor.PostSubmitTestPath,
+		}
+
+		scope.Debugf("Getting test results for org %s", repo.OrgLogin)
+		var completedTests = make(map[string]bool)
+		ctLock := sync.RWMutex{}
+		wg := sync.WaitGroup{}
+		wg.Add(1)
+		go func() {
+			ctLock.Lock()
+			defer ctLock.Unlock()
+			err := ss.mgr.store.QueryTestResultByDone(ss.ctx, repo.OrgLogin, repo.RepoName,
+				func(result *storage.TestResult) error {
+					completedTests[result.RunPath] = true
+					return nil
+				})
+			if err != nil {
+				scope.Warnf("Unable to fetch previous tests: %s", err)
+			}
+			wg.Done()
+		}()
+		prPaths := g.GetAllPostSubmitTestChan(ss.ctx).WithBuffer(100)
```

Comment:
prPaths should probably be renamed, since it doesn't directly have to do with pull requests.